src/field/fieldperp.cxx

```diff
@@ -33,7 +34,7 @@
 #include <boutexception.hxx>
 #include <msg_stack.hxx>
 
-FieldPerp::FieldPerp(Mesh *localmesh) {
+FieldPerp::FieldPerp(Mesh *localmesh) : Field(localmesh), yindex(-1) {
   // Get mesh size
   fieldmesh = localmesh;
```

Comment:
fieldmesh should already be set.

Reply:
Ah yes thanks, it didn't used to initialise Field automatically.

Reply:
Should be addressed now